console/docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+---
+services:
+
+  caddy:
+    image: lucaslorentz/caddy-docker-proxy:ci-alpine
```

Comment:
```suggestion
    image: caddy:latest
```

I think it's worth using the official caddy [image](https://hub.docker.com/_/caddy).

@sbusso what do you think?

Reply:
@vitabaks They work differently. The docker proxy is a popular caddy plugin to dynamically configure caddy using service labels, like Traefik. Caddy alone would require a configuration file.

Reply:
Thanks for the clarification. Then maybe the tag should be replaced with `latest`? Why is "ci-alpine" used?

Reply:
This image is the smallest image using `alpine` and this will also be latest.